test/170_c_centering.txt

```diff
@@ -0,0 +1,9 @@
+:SUFFIX
+:ARGS
+-s 100 -a hcvc -c
```

Comment:
Would it be possible to just add a quoted space here, so as to avoid the "hack" in the test runner?
```suggestion
-s 100 -a hcvc -c " "
```